database/rrdhost.c

```diff
@@ -687,6 +689,7 @@ RRDHOST *rrdhost_find_or_create(
            , rrdpush_api_key
            , rrdpush_send_charts_matching
            , system_info);
+        sql_store_host_system_info(&host->host_uuid, system_info);
```

Comment:
Is this intentional? `sql_store_host_system_info` is already called from `rrdhost_update`.

Reply:
Will remove it